command/namespace_scan.go

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
I'd add our copyright header as well. Scan is net-new to OpenBao, though I understand the structure of commands is very similar across all of them.